src/Sulu/Bundle/AdminBundle/Resources/js/containers/Badge/stores/BadgeStore.js

```diff
@@ -82,6 +85,15 @@ export default class BadgeStore {
     }
 
     load = debounce(() => {
+        let route: ?Route = this.router.route;
```

Comment:
would extract this into something like a `isChildRoute` function to give some context about what it is doing 😇